### DISCUSSION TITLE
Change local hostname and https from http

### DISCUSSION THREAD

**Discussion by dsauna** [vercel/next.js#19429] [Help]
Hi All, 

With CRA to change my hostname and set https locally I can simply add HOST=projectName.local.proxy HTTPS=true to my .env.development. Is there a way that I can do something like that with next?

Thanks in advance.

**Comment by jamesmosier** [comment]
There is no built-in way to do this in Next currently. There was another discussion that has some potential solutions though: https://github.com/vercel/next.js/discussions/10935

I believe there are some other discussions/issues that mention this although I wasn't able to find them.

**Reply by dsauna** [reply]
Thanks so much. I might give that a try.
